ern-api-gen/src/apigen.js

```diff
@@ -30,7 +29,7 @@ import {
 export async function generateApi (options: Object) {
   let config = normalizeConfig(options)
 
-  const outFolder = `${process.cwd()}/${config.moduleName}`
+  const outFolder = path.join(process.cwd(), `${config.moduleName}`)
```

Comment:
can change `${config.moduleName}` to `config.moduleName`

Reply:
👍